### PR TITLE
Refine documentation related to logging

### PR DESCRIPTION
**Pull request by pellared**
The log directory is optional. It may be worth describing it (as on `main`) as the log directory may bring problems on container deployment (where containers can have little storage assigned).

### CODE REVIEW

**Review by Kielek — APPROVED** [review]

File: docs/advanced-config.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Use these environment variables to configure the tracing library:
...
 | `SIGNALFX_TRACE_STARTUP_LOGS` | Enable to activate diagnostic log at stratup. | `true` |
-| `SIGNALFX_TRACE_SAMPLING_RULES` | Comma separated list of sampling rules taht enabled custom sampling rules based on regular expressions. The rule is matched in order of specification. The first match in a list is used. The item "sample_rate" is required in decimal format. The item "service" is optional in regular expression format, to match on service name. The item "name" is optional in regular expression format, to match on operation name. | `'[{"sample_rate":0.5, "service":"cart.*"}],[{"sample_rate":0.2, "name":"http.request"}]'` |
+| `SIGNALFX_TRACE_SAMPLING_RULES` | Comma separated list of sampling rules that enabled custom sampling rules based on regular expressions. The rule is matched in order of specification. The first match in a list is used. The item "sample_rate" is required in decimal format. The item "service" is optional in regular expression format, to match on service name. The item "name" is optional in regular expression format, to match on operation name. | `'[{"sample_rate":0.5, "service":"cart.*"}],[{"sample_rate":0.2, "name":"http.request"}]'` |
```
Comment (theletterf) [inline docs/advanced-config.md:54]:
```suggestion
| `SIGNALFX_TRACE_SAMPLING_RULES` | Comma separated list of sampling rules that enable custom sampling rules based on regular expressions. The rule is matched in order of specification. The first match in a list is used. The item "sample_rate" is required in decimal format. The item "service" is optional in regular expression format, to match on service name. The item "name" is optional in regular expression format, to match on operation name. | `'[{"sample_rate":0.5, "service":"cart.*"}],[{"sample_rate":0.2, "name":"http.request"}]'` |
```

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -36,17 +36,22 @@ like [Process Explorer](https://docs.microsoft.com/en-us/sysinternals/downloads/
 On Linux you can simply run: `cat /proc/<pid>/environ`
```
Comment (theletterf) [inline docs/troubleshooting.md:36]:
```suggestion
On Linux, run: `cat /proc/<pid>/environ`
```

File: docs/troubleshooting.md [hunk 3]
```diff
@@ -36,17 +36,22 @@ like [Process Explorer](https://docs.microsoft.com/en-us/sysinternals/downloads/
...
 where `<pid>` is the Process ID.
 
-If none of the suggestions above solves your issue, detailed logs are necessary.
-Follow the steps below to get detailed logs.
+If all settings look fine and none of the suggestions above solves your issue,
+detailed logs are necessary. Follow the steps below to get detailed logs.
```
Comment (theletterf) [inline docs/troubleshooting.md:40]:
```suggestion
Enable debug logging If none of the suggestions solved your issue. Follow these steps to enable debug logging for .NET instrumentation:
```

File: docs/troubleshooting.md [hunk 4]
```diff
@@ -36,17 +36,22 @@ like [Process Explorer](https://docs.microsoft.com/en-us/sysinternals/downloads/
...
 
 Set the environment variable `SIGNALFX_TRACE_DEBUG` to `true` before
 the instrumented process starts.
 By default, the library writes the log files under the below predefined locations.
 If needed, change the default location by updating the environment variable
 `SIGNALFX_TRACE_LOG_DIRECTORY` to an appropriate path.
-On Linux, the default log location is `/var/log/signalfx/dotnet/`
+
+On Linux, the default log location is `/var/log/signalfx/dotnet/`. If it does not
+exist, then run `/opt/signalfx/createLogPath.sh` to create it with appropriate permissions.
```
Comment (theletterf) [inline docs/troubleshooting.md:49]:
```suggestion
exist,  run `/opt/signalfx/createLogPath.sh` to create it with the required permissions.
```

**Review by theletterf — CHANGES_REQUESTED** [review]
Just a few nits!

**Review by theletterf — APPROVED** [review]

**Pull request merged**